README.md

```diff
@@ -42,83 +34,110 @@ Run `source environment`  now and whenever these environment files are modified.
-  and set the environment variable `DSS_GS_BUCKET_TEST_FIXTURES` to the name of that bucket.
+
+1.  Follow the instructions in https://cloud.google.com/sdk/downloads to get the `gcloud` command line utility.
+
+2.  In the [Google Cloud Console](https://console.cloud.google.com/), select the correct Google user account on the top right and the correct GCP project in the drop down in the top center. Go to "IAM & Admin", then "Service accounts", then click "Create service account" and select "Furnish a new private key". Under "Roles" select "Project – Owner", "Project – Service Account Actor" and "Cloud Functions – Cloud Function Developer". Create the account and download the service account key JSON file.
+
+3.  In `environment.local`, set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the path of the service account key JSON file.
+
+4.  Choose a region that has support for Cloud Functions and set `GCP_DEFAULT_REGION` to that region. See https://cloud.google.com/about/locations/ for a list of supported regions.
+
+5.  Run `gcloud auth activate-service-account --key-file=/path/to/service-account.json`.
+
+6.  Run `gcloud config set project PROJECT_ID` where PROJECT_ID is the ID, not the name (!) of the GCP project you selected earlier.
+
+7.  Enable required APIs: `gcloud service-management enable cloudfunctions.googleapis.com`; `gcloud service-management enable runtimeconfig.googleapis.com`
+
+8.  Generate OAuth application secrets to be used for your instance: 
+
+    1) Go to https://console.developers.google.com/apis/credentials (you may have to select Organization and Project again)
+    
+	2) Click *Create Credentials* and select *OAuth client*
+    
+	3) For *Application type* choose *Other*  
+    
+	4) Under application name, use `hca-dss-` followed by the stage name i.e., the value of DSS_DEPLOYMENT_STAGE. This is a convention only and carries no technical significance.
+    
+	5) Click *Create*, don't worry about noting the client ID and secret, click *OK*
+    
+	6) Click the edit icon for the new credentials and click *Download JSON*
+    
+	7) Place the downloaded JSON file into the project root as `application_secrets.json`
+
+9.  Create a Google Cloud Storage bucket and in `environment.local`, set the environment variable `DSS_GS_BUCKET` to the name of that bucket. Make sure the bucket region is consistent with `GCP_DEFAULT_REGION` in `environment.local`.
+
+10. If you wish to run the unit tests, you must create two more buckets, one for test data and another for test fixtures, and set the environment variables `DSS_GS_BUCKET_TEST` and `DSS_GS_BUCKET_TEST_FIXTURES` to the names of those buckets.
+
+Hint: To create GCS buckets from the command line, use `gsutil mb -c regional -l REGION gs://BUCKET_NAME/`.
 
 ##### Azure
-* Set the environment variables `AZURE_STORAGE_ACCOUNT_NAME` and `AZURE_STORAGE_ACCOUNT_KEY`.
+
+1. Set the environment variables `AZURE_STORAGE_ACCOUNT_NAME` and `AZURE_STORAGE_ACCOUNT_KEY`.
```

Comment:
Why even include an Azure section in the readme at this point?
I don't believe we have any support for it at all, currently.